RecoMTD/Configuration/python/RecoMTD_EventContent_cff.py

```diff
@@ -1,20 +1,22 @@
 import FWCore.ParameterSet.Config as cms
 
-#AOD
-RecoMTDAOD = cms.PSet(
+#FEVT content
+RecoMTDFEVT = cms.PSet(
     outputCommands = cms.untracked.vstring(
         'keep *_trackExtenderWithMTD_*_*',
         'keep *_mtdTrackQualityMVA_*_*')
 )
 
 #RECO content
 RecoMTDRECO = cms.PSet(
-    outputCommands = cms.untracked.vstring() 
+    outputCommands = cms.untracked.vstring()
 )
-RecoMTDRECO.outputCommands.extend(RecoMTDAOD.outputCommands)
+RecoMTDRECO.outputCommands.extend(RecoMTDFEVT.outputCommands)
```

Comment:
Should this be done in the opposite way, e.g. define RECO content, and add to FEVT? With this way, one can add content in FEVT-only easier in the future.

Reply:
I do not see a short term use case like that, but I agree on the general principle